composer/installed.json

```diff
@@ -437,7 +437,7 @@
         },
         "dist": {
             "type": "zip",
-            "url": "https://api.github.com/repos/JamesHeinrich/getID3/zipball/178e55bc05bc78c764ae0a096c4caacf118ab12a",
+            "url": "https://api.github.com/repos/JamesHeinrich/getID3/zipball/824825682a3229db8d5f1bd8c2c0cb6aae7c1ec2",
```

Comment:
Like I said in the 2015-05-05 comment: "I don't know what's up with it changing the URLs to various things in composer.lock. composer moves in mysterious ways."

I think it's something like, even with `--no-update` on the `remove` and `require` commands, it finds out that there are newer builds of some things and updates the `.lock` file, though `--no-update` stops it actually downloading the newer builds.

If you know a better way to add something without causing changes, by all means, let me know. Again all I did was run the commands listed in that earlier comment, you can reproduce the commit simply by doing that.